test/utils/SAVAXDepositHelper.ts

```diff
@@ -0,0 +1,56 @@
+import { ethers, network } from "hardhat";
+import { Contract } from "ethers";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/dist/src/signers";
+import { assert } from "../helpers/assertions";
+import { CHAINID, SAVAX_ADDRESS } from "../../constants/constants";
+
+import AvaxRibbonThetaVaultLogic from "../../deployments/avax/RibbonThetaVaultLogic.json";
+import RibbonThetaVaultSAVAXCall from "../../deployments/avax/RibbonThetaVaultSAVAXCall.json";
+
+describe("SAVAXDepositHelper", () => {
+  let sAVAXDepositHelper: Contract;
+  let sAVAXVault: Contract;
+  let signer: SignerWithAddress;
+
+  before(async () => {
+    await network.provider.request({
+      method: "hardhat_reset",
+      params: [
+        {
+          forking: {
+            jsonRpcUrl: process.env.AVAX_URI,
+            blockNumber: 11518500,
```

Comment:
I don't see a similar block number forked in the existing tests, would increase test speed if we could share the same block numbers :)

Reply:
Yeah - will fix but it will break the other tests unless the blocknumber >1 day.

Reply:
Fixed